src/misc/messages.cpp

```diff
@@ -115,16 +116,18 @@ class Message {
 	}
 };
 
-static std::unordered_map<std::string, Message> messages;
-static std::deque<std::string> messages_order;
+static std::unordered_map<std::string, Message> messages = {};
+static std::deque<std::string> messages_order            = {};
```

Comment:
Do you really need `deque`, btw? I'd prefer using `vector` as the default array-like container, and only use these specialisations if there are very good reasons (like huge performance improvements when prepending many elements at the front when using `deque`).

Reply:
`std::vector` is enough here. Changed.